examples/CMakeLists.txt

```
@@ -453,6 +453,24 @@ add_impactx_test(solenoid.MADX.py
     OFF  # no plot script yet
 )
 
+# Pole-face rotation ###################################################
+#
+# w/o space charge
+add_impactx_test(rotation
+    examples/rotation/input_rotation.in
+      ON   # ImpactX MPI-parallel
+      OFF  # ImpactX Python interface
+    examples/rotation/analysis_rotation.py
+    OFF  # no plot script yet
+)
+add_impactx_test(rotation.py
+    examples/rotation/run_rotation.py
+      OFF  # ImpactX MPI-parallel
```

I noticed the non-Python version uses MPI, the Python version does not. We can set this to use MPI as well, if you like:
```suggestion
      ON   # ImpactX MPI-parallel
```
Physically, the results should be the same either way :)